chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CertifiedProduct.java

```diff
@@ -69,6 +70,36 @@
     @XmlElement(required = false, nillable = true)
     private Boolean curesUpdate;
 
+    /**
+     * The first year that the listing is eligible Real World Testing
+     */
+    @XmlElement(required = false, nillable = true)
```

Comment:
Why do we need these elements here? I thought they might only be in the CertifiedProductSearchDetails class.